app/src/main/java/com/example/fittracker/UserRoomDatabase.java

```diff
@@ -5,7 +5,7 @@
 import android.arch.persistence.room.RoomDatabase;
 import android.content.Context;
 
-@Database(entities = {User.class}, version = 1)
+@Database(entities = {User.class}, version = 2)
```

Comment:
use constants pls. You should have a global property with the DB version, you can declare this globalProperty in the gradle since it will be general for the whole app

Reply:
Done